app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -29,6 +29,7 @@ def create_with_dynflow
             @taxonomy = Organization.new(resource_params)
             sync_task(::Actions::Katello::Organization::Create, @taxonomy)
             @taxonomy.reload
+            session[:organization_id] = @taxonomy.id
```

Comment:
@ehelms, There is a [PR](https://github.com/theforeman/foreman/pull/4118) for Foreman as well

> IMHO it would be better to redirect to the right action in the controller, and let the controller take care of this action.

@ShimShtein, that would mean creating a new action in the controller and additional redirect - not sure if the complexity is worth it, but I can draft a solution